src/test/java/com/example/rets_api/repository/UtilsTest.java

```diff
@@ -151,6 +153,8 @@ public static PropertyDTO createPropertyDTOWithBasicFields(){
             .propertyTypeTownHouse(true)
             .financialData(createFinancialDataDTO())
             .roomList(asList(createRoomDTO(RoomType.MASTER_BEDROOM)))
+            .viewData(createViewDataDTO())
+            .roomList(asList(createRoomDTO(RoomType.LIVING_ROOM)))
```

Comment:
Here the roomList was replaced, need to be like this .roomList(asList(createRoomDTO(RoomType.MASTER_BEDROOM), createRoomDTO(RoomType.LIVING_ROOM))), check the line 38 for reference